content/blog/announce-vetiver/index.Rmd

```diff
@@ -0,0 +1,117 @@
+  trained model.
+
+photo:
+  url: https://unsplash.com/photos/3C5ZfCLSGC4
+  author: "21 swan"
+
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [tidymodels, vetiver]
+---
+
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+```{r include=FALSE, message=FALSE}
+library(ggplot2)
+theme_set(theme_minimal())
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>", message = FALSE)
+options(cli.width = 70, width = 70)
+```
+
+
+We are thrilled to announce the release of [vetiver](https://vetiver.rstudio.com/), a framework for MLOps tasks in R and Python! The goal of vetiver is to provide fluent tooling to **version**, **share**, **deploy**, and **monitor** a trained model. If you like perfume or candles, you may recognize this ingredient; vetiver, also known as the "oil of tranquility", is used as a stabilizing ingredient in perfumery to preserve more volatile fragrances.
+
+You can install the released version of vetiver for R from [CRAN](https://cran.r-project.org/package=vetiver):
+
+```r
+install.packages("vetiver")
+```
+
+You can install the released version of vetiver for Python from [PyPI](https://pypi.org/project/vetiver/):
+
+```python
+pip install vetiver
+```
+
+We are sharing more about what vetiver is and how it works over [on the RStudio blog](https://www.rstudio.com/blog/announce-vetiver/) so check that out, but we want to share here as well!
+
+## Train a model
+
+For this example, let’s work with data on everyone's favorite dataset on fuel efficiency for cars to predict miles per gallon. In R, we can train a decision tree model to predict miles per gallon using a [tidymodels](https://www.tidymodels.org/) workflow:
+
+```{r}
+#| message: false
+library(tidymodels)
+
+car_mod <-
+    workflow(mpg ~ ., decision_tree(mode = "regression")) %>%
+    fit(mtcars)
+```
+
+In Python, we can train the same kind of model using [scikit-learn](https://scikit-learn.org/):
+
+```{python}
+from vetiver.data import mtcars
+from sklearn import tree
+car_mod = tree.DecisionTreeRegressor().fit(mtcars, mtcars["mpg"])
+```
+
+For both R and Python, the `car_mod` object is a fitted model, with parameters estimated using our training data `mtcars`.
+
+
+## Create a vetiver model
+
+We can create a `vetiver_model()` in R or `VetiverModel()` in Python from the trained model; a vetiver model object collects the information needed to store, version, and deploy a trained model.
+
+```{r}
```

Comment:
Maybe not for this blog post, but I'm wondering if it would be helpful to have a language indicator for code chunks in posts where we're using more than one language (e.g. Garrick described an Rmd implementation here: https://rstudio.slack.com/archives/C9NK7JY5B/p1654187053132789).

Once we move to Quarto, tabsets would be the obvious way to go. Basically bringing it up here to see if you think it'd be worthwhile, too.

Reply:
I think this would be useful going forward, we have been spoiled with quarto in other formats!